web-wallet/src/lib/dusk/components/ErrorDetails/ErrorDetails.svelte

```diff
@@ -1,31 +1,31 @@
-<svelte:options immutable={true}/>
+<svelte:options immutable={true} />
 
 <script>
-	import { makeClassName } from "$lib/dusk/string";
+  import { makeClassName } from "$lib/dusk/string";
 
-	/** @type {String | Undefined} */
-	export let className = undefined;
+  /** @type {String | Undefined} */
+  export let className = undefined;
 
-	/** @type {Error | null} */
-	export let error;
+  /** @type {Error | null} */
+  export let error;
 
-	/** @type {String} */
-	export let summary;
+  /** @type {String} */
+  export let summary;
 
-	$: classes = makeClassName([
-		"dusk-error-details",
-		"dusk-error-details__details",
-		className
-	]);
+  $: classes = makeClassName([
+    "dusk-error-details",
+    "dusk-error-details__details",
+    className,
+  ]);
 </script>
 
 {#if error}
-	<details class={classes}>
-		<summary class="dusk-error-details__summary">
-			{summary}
-		</summary>
-		<pre class="dusk-error-details__error">
+  <details class={classes}>
+    <summary class="dusk-error-details__summary">
+      {summary}
+    </summary>
+    <pre class="dusk-error-details__error">
 			{error.message}
```

Comment:
This is an odd issue that @deuch13 had experienced in his PR as well. Looking at the file, it looks fine

<img width="472" alt="Screenshot 2024-02-28 at 11 47 09" src="https://github.com/dusk-network/rusk/assets/34094428/fee1193f-730a-4f0c-9df7-f3efb161d576">

however, if you remove the white space and press enter, you will see GitHub showing the change in the file, as follows:

<img width="1943" alt="Screenshot 2024-02-28 at 11 46 09" src="https://github.com/dusk-network/rusk/assets/34094428/482727d7-5e05-43a6-b538-71ac8d1af1ce">

Which resolves the issue.


Seems like it's a character that is encoded differently (and rendered differently in different environments) – for example, if you try to copy and paste the whitespace in a text field, you will see there is a difference, as shown in the video below:

https://github.com/dusk-network/rusk/assets/34094428/3991deee-2193-46b1-bc76-66448350f8ad

It looks like it is a bug with the linter... doesn't happen each time, but this is the second time we are seeing this.

Reply:
> It looks like it is a bug with the linter... doesn't happen each time, but this is the second time we are seeing this.

Was the other time also about white spaces inside a tag?

(this can be the reason: how to interpret it and whether such white space should be left alone or not)

Reply:
No, it was white space in a variable definition. `let   variable`